config/platform/belgie.php

```diff
@@ -90,12 +91,10 @@
                 ],
                 'deliveryTypes'   => [
                     DeliveryOptions::DELIVERY_TYPE_STANDARD_NAME,
+                    DeliveryOptions::DELIVERY_TYPE_PICKUP_NAME,
                 ],
                 'shipmentOptions' => [
-                    'insurance' => [
-                        0,
-                        52000,
-                    ],
+                    'insurance' => false,
```

Comment:
waarom false? dit zou `[0]` moeten zijn als dat de enige geaccepteerde waarde is